modules/terraform/show.go

```diff
@@ -0,0 +1,19 @@
+package terraform
+
+import (
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// Show runs terraform show with the given options and returns stdout/stderr.
+// This will fail the test if there is an error in the command.
+func Show(t testing.TestingT, options *Options) string {
```

Comment:
Is there a reason why this isn't returning the `Plan` struct?

Reply:
I can't answer for @pysysops but think this makes sense in line with other functions such as `terraform.Init`?
https://github.com/gruntwork-io/terratest/blob/56fc21222eaa395af081e5d629e5b81da49098b8/modules/terraform/init.go#L10

At present the `Plan` struct gets created like this:
https://github.com/gruntwork-io/terratest/blob/a97c8ce784c19935fd696b30b6becd3def889abf/modules/terraform/show_test.go#L38-L42

...so perhaps we could answer your comment by adding some syntactic sugar to wrap calling `terraform.Show` and `jsonplan.Unmarshal`?

Personally I prefer the current method, as I think it makes it clearer that we are taking Terraform's JSON plan output and turning it into a format that is internal to Terratest.

However, it's ultimately a trade-off between writeability (with syntactic sugar) and readability (current method), so I'm happy to implement this however maintainers would prefer.

Reply:
This makes sense to me. Thanks for the clarifying argument in favor of returning string.